src/components/command-bar/commands/search/unified-search/components/dialog-body/chat.module.css

```diff
@@ -0,0 +1,23 @@
+.ask {
```

Comment:
For styling 
![CleanShot 2023-09-08 at 11 32 23@2x](https://github.com/hashicorp/dev-portal/assets/26389321/7c7fb7e0-8ba3-4ea3-b87f-c914b630d89e)